docs/RestoreDatabase.md

```diff
@@ -0,0 +1,210 @@
+# Restore database guide
+
+This guide explains how to restore ScalarDB/ScalarDL data on a Kubernetes environment. We assume you use the managed databases provided by cloud services as a backend database of ScalarDB/ScalarDL.
```

Comment:
```suggestion
This guide explains how to restore ScalarDB or ScalarDL data in a Kubernetes environment. Please note that this guide assumes that you are using a managed database from a cloud services provider as the backend database for ScalarDB or ScalarDL.
```

Reply:
As I mentioned in another comment, I think it would be better to describe `databases that are used by ScalarDB or ScalarDL` instead of `databases that use ScalarDB or ScalarDL`.

What do you think?

Reply:
I agree. To avoid passive voice, how about we describe that part as `databases that ScalarDB or ScalarDL uses` instead of `databases that are used by ScalarDB or ScalarDL`?

```suggestion
This guide explains how to restore databases that ScalarDB or ScalarDL uses in a Kubernetes environment. Please note that this guide assumes that you are using a managed database from a cloud services provider as the backend database for ScalarDB or ScalarDL.
```